src/react/components/pages/predict/predictPage.tsx

```diff
@@ -250,28 +250,32 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                                                 <h5 className="model-selection-header">
                                                     {strings.predict.selectModelHeader}
                                                 </h5>
-                                                <tr>
-                                                    <h6 className="model-selection-info-header" >
-                                                        <span className="model-selection-info-key">
-                                                            {strings.predict.modelIDPrefix}
-                                                        </span>
-                                                        <span title={mostRecentModel.modelInfo.modelId} className="model-selection-info-value">
-                                                            {mostRecentModel.modelInfo.modelId}
-                                                        </span>
-                                                    </h6>
-                                                </tr>
-                                                <tr>
-                                                    {mostRecentModel.modelInfo.modelName &&
-                                                        <h6 className="model-selection-info-header" >
-                                                            <span className="model-selection-info-key">
-                                                                {strings.predict.modelNamePrefix}
-                                                            </span>
-                                                            <span title={mostRecentModel.modelInfo.modelName} className="model-selection-info-value">
-                                                                {mostRecentModel.modelInfo.modelName}
-                                                            </span>
-                                                        </h6>
-                                                    }
-                                                </tr>
+                                                <table>
+                                                    <tbody>
+                                                        <tr>
+                                                            <th className="model-selection-info-header" >
```

Comment:
td looks better

Reply:
isn't so bold

Reply:
![image](https://user-images.githubusercontent.com/60453211/88097265-bf8f1600-cb4c-11ea-97b5-1be81e80a481.png)
vs
![image](https://user-images.githubusercontent.com/60453211/88097302-ccac0500-cb4c-11ea-90da-14789795134d.png)

Reply:
also add class p-0 to td to remove the extra padding